src/jwt-browser.js

```diff
@@ -1,5 +1,15 @@
+import {Plugin} from 'fusion-core';
+
 export default function({secret} = {}) {
   if (__DEV__ && secret) {
-    throw new Error('Do not pass JWT session secret to the client');
+    throw new Error(
+      'Do not pass JWT session secret to the client. Try: `app.plugin(JWTSession, __NODE__ && {...})`'
```

Comment:
Thanks for this change!  +1 for error messages providing helpful suggestions where possible.